src/redist/targets/GenerateMSIs.targets

```diff
@@ -283,49 +280,36 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="GenerateManifestsMsi"
-        DependsOnTargets="GenerateLayout;AcquireWix;MsiTargetsSetupInputOutputs;SetSdkBrandingInfo"
-        Condition="$(ProductMonikerRid.StartsWith('win')) And '$(Architecture)' != 'arm'"
-        Inputs="@(BundledManifests);$(ManifestsGenerateMsiPowershellScript)"
-        Outputs="$(ManifestsMSIInstallerFile)">
-
+  <Target Name="GenerateWorkloadManifestsWxs">
     <PropertyGroup>
-      <ManifestsGenerateMsiPowershellScript>$(MSBuildThisFileDirectory)packaging/windows/clisdk/generatemanifestsmsi.ps1</ManifestsGenerateMsiPowershellScript>
+      <WorkloadManifestsWxsPath>$(IntermediateOutputPath)WorkloadManifests.wxs</WorkloadManifestsWxsPath>
+      <WorkloadManifestsWxsContent>
+<![CDATA[
```

Comment:
nit: might be cleaner to have the content as a separate file and follow the search and replace pattern to replace msbuild properties [we use in other places](https://github.com/dotnet/installer/blob/main/src/redist/targets/GeneratePKG.targets#L46)

Reply:
It's not just a simple string replacement though, it needs to add an MsiPackage element for each `BundledManifests` item.  So the replacement string would still have to be generated in a similar way to how this is generating the whole file.